addons/l10n_ec/data/l10n_ec_identification_type_data.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <data>
+        <record id="ec_move_dt_01" model="l10n_ec.identification.type">
+            <field name="code">01</field>
+            <field name="name">COMPRA - RUC</field>
```

Comment:
Why not use the identification type that we have on the partner already?

Reply:
Removed, python implementation